src/http.js

```diff
@@ -72,9 +72,10 @@ export default class HTTP {
    * @param  {String} url               The URL.
    * @param  {Object} [options={}]      The fetch() options object.
    * @param  {Object} [options.headers] The request headers object (default: {})
+   * @param  {Object} [options.retry]   Number of retries (default: 1)
    * @return {Promise}
    */
-  request(url, options={headers:{}}) {
+  request(url, options={headers:{}, retry: 1}) {
```

Comment:
I usually prefer calling specific behaviors explicitly, but in this case we could probably live with a songle retry performed by default. Your call.